README.md

```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
 # hw01
 Repository for first CFSS homework assignment
```

Comment:
header “hw01; Repository for first CFSS homework assignment” might be deleted, and your name could be added when pulling request.